scripts/zephyr_module.py

```diff
@@ -378,14 +378,16 @@ def west_projects():
 
     from packaging import version
     try:
-        manifest = Manifest.from_file()
+        if not manifest:
+            manifest = Manifest.from_file()
         if version.parse(WestVersion) >= version.parse('0.9.0'):
             projects = [p for p in manifest.get_projects([])
                         if manifest.is_active(p)]
         else:
             projects = manifest.get_projects([])
         manifest_file = manifest.path
-        return {'manifest': manifest_file, 'projects': projects}
+        return {'manifest_file': manifest_file,
+                'projects': [p.posixpath for p in projects]}
```

Comment:
as this is no longer the `projects` as seen from `manifest` object, let's avoid the same confusion we just had with `manifest` being a filepath, and instead call this `projects_path`.
```suggestion
                'projects_path': [p.posixpath for p in projects]}
```

Reply:
Fixed